functional-samples/cookbook.wasm-helloworld-print-nomodule/wasm/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+extern crate wasm_bindgen;
+
+use wasm_bindgen::prelude::*;
+use web_sys::console;
+
+// will be called when the wasm module is loaded
+// https://rustwasm.github.io/docs/wasm-bindgen/reference/attributes/on-rust-exports/start.html
+#[wasm_bindgen(start)]
+pub fn main() {
+    console::log_1(&"[from wasm] Inited.".into());
+}
+
+#[wasm_bindgen]
+pub fn print() {
+    console::log_1(&"[from wasm] Hello World!".into());
+}
+
+#[wasm_bindgen]
+pub fn print_with_value(value: &str) {
+    // with format! macro
+    console::log_1(&format!("[from wasm] Hello {}", value).into());
```

Comment:
We can probably remove this line? The point of this tutorial is to teach using WASM in extensions, not Rust, so I don't think we need to show `format!`.